src/app/ui/app/App.tsx

```diff
@@ -62,12 +74,23 @@ const App = ({ starterVariant, isWelcomeScreenClosed, children }: AppProps) => {
           <WelcomeScreen closed={welcomeScreen.welcomeScreenClosed!} />
         </Box>
       </StyledSlide>
-      <WidgetContainer
-        welcomeScreenClosed={welcomeScreen.welcomeScreenClosed!}
-        className="widget-container"
+      <Stack
+        direction="row"
+        justifyContent="center"
+        alignItems="start"
+        spacing={4}
+        paddingTop={3.5}
       >
-        {children}
-      </WidgetContainer>
+        {!isLearnPage && !hasTheme ? (
```

Comment:
we might want to change the location of the navbar component to the `widget` to avoid these booleans. What do you think @oktapodia ?

Reply:
we can technically do it, although widget component will not be a single responsibility component then which is usually good

Reply:
Let's think about it later please because I am already changing a lot of structure in a PR soon, would love to avoid more conflicts than we will have